content/en/blog/_posts/2021-07-19-alpha-swap-support.md

```diff
@@ -0,0 +1,139 @@
+The admin can specify the node's behaviour in the presence of swap memory by
+setting `memorySwap.swapBehavior`.
+
+This is possible through [the addition of a `memory_swap_limit_in_bytes`
+field](https://github.com/kubernetes/enhancements/blob/9d127347773ad19894ca488ee04f1cd3af5774fc/keps/sig-node/2400-node-swap/README.md#cri-changes)
+to the container runtime interface (CRI). The kubelet's config will control how
+much swap memory the kubelet instructs the container runtime to allocate to
+each container via the CRI. The container runtime will then write the swap
+settings to the container level cgroup.
+
+## How do I use it?
+
+On a node where swap memory is already provisioned, Kubernetes use of swap on a
+node can be enabled by enabling the `NodeSwap` feature gate on the kubelet, and
+disabling the `failSwapOn` [configuration setting](/docs/reference/config-api/kubelet-config.v1beta1/#kubelet-config-k8s-io-v1beta1-KubeletConfiguration)
+or the `--fail-swap-on` command line flag.
+
+A user can also optionally configure `memorySwap.swapBehavior` in order to
+specify how a node will use swap memory. For example,
+
+```yaml
+memorySwap:
+  swapBehavior: LimitedSwap
+```
+
+The available configuration options for `swapBehavior` are:
+
+- `LimitedSwap` (default): Kubernetes workloads are limited in how much swap
+  they can use. Workloads on the node not managed by Kubernetes can still swap.
+- `UnlimitedSwap`: Kubernetes workloads can use as much swap memory as they
+  request, up to the system limit.
+
+If configuration for `memorySwap` is not specified and the feature gate is
+enabled, by default the kubelet will apply the same behaviour as the
+`LimitedSwap` setting.
+
+The behaviour of the `LimitedSwap` setting depends if the node is running with
+v1 or v2 of control groups (also known as "cgroups"):
+
+- **cgroupsv1:** Kubernetes workloads can use any combination of memory and
+  swap, up to the pod's memory limit, if set.
+- **cgroupsv2:** Kubernetes workloads cannot use swap memory.
+
+### Caveats
+
+Having swap available on a system reduces predictability. Swap's performance is
+worse than regular memory, sometimes by many orders of magnitude, which can
+cause unexpected performance regressions. Furthermore, swap changes a system's
+behaviour under memory pressure, and applications cannot directly control what
+portions of their memory usage are swapped out. Since enabling swap permits
+greater memory usage for workloads in Kubernetes that cannot be predictably
+accounted for, it also increases the risk of noisy neighbours and unexpected
+packing configurations, as the scheduler cannot account for swap memory usage.
+
+Hence, we do not recommend the use of swap for certain performance-constrained
+workloads. Cluster administrators and developers should benchmark their nodes
+and applications before using swap in production scenarios, and [we need your
+help](#how-do-i-get-involved) with that!
+
+## What's next?
```

Comment:
To avoid readers confusing this with documentation where this heading has a specific meaning, I'd write:
```suggestion
## Looking ahead
```